tdodson/js-dev/theme.js

```diff
@@ -0,0 +1,6 @@
+jQuery(document).ready(function($) {
+    var $isEditorialArticle = $('body').hasClass('single-bueditorialarticle');
+    if ($isEditorialArticle = true) {
```

Comment:
It appears you’re trying to validate a css class string as a true/false boolean value.